### PR TITLE
feat/polynomial-v2-vaults

### PR DESCRIPTION
**Pull request by KngZhi**
Update on V2 vaults

Output combined v1 and v2:

```
--- optimism ---
SETH                      423.28 k
SUSD                      311.33 k
SBTC                      0.9016052760937305
Total: 734.61 k

--- tvl ---
SETH                      423.28 k
SUSD                      311.33 k
SBTC                      1
Total: 734.61 k

------ TVL ------
optimism                  734.61 k

total                    734.61 k
```

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/polynomial exports TVL: 
        
 
 
    optimism                  728.99 k
    
    total                    728.99 k

File: projects/polynomial/index.js [hunk 1]
```diff
@@ -83,16 +101,85 @@ async function tvl (timestamp, ethBlock, chainBlocks) {
...
 
+function tvlSummup(contract) {
+  const { totalFund, withdrawal, deposit } = contract
+
+  return new BigNumber(totalFund)
+    .plus(new BigNumber(deposit))
+    .minus(new BigNumber(withdrawal))
+}
+async function tvl2(block) {
+  const ENDPOINT = 'https://earn-api.polynomial.fi/vaults'
```
Comment (g1nt0ki) [inline projects/polynomial/index.js:132]: is there a way to look up vaut list on chain?

Comment (KngZhi) [reply projects/polynomial/index.js:134]: We can't get vault list on-chain.

**Comment by KngZhi** [comment]
Hi @g1nt0ki sorry for the late reply, this PR is ready to go.

Previously had some details needed to settle down within the team.

**Comment by llamatester** [comment]
The adapter at projects/polynomial exports TVL: 
        
 
 
    optimism                  751.25 k
    
    total                    751.25 k

**Pull request merged**